README.md

```diff
@@ -1,3 +1,13 @@
+# Aptogotchi (Beginner)
+
+This is the beginner-level dApp for the Aptogotchi Tutorial demos hosted on [Builder Hub](https://github.com/aptos-labs/aptos-builder-hub).
+
+Aptogotchi is an educational and discovery surface of the demos we build. The goal is removing the barriers for developers to start building on Aptos.
```

Comment:
isn't this the description of builder hub? aptogotchi should be the demo on the top of the hub